Resources/Prototypes/Entities/Mobs/Player/guardian.yml

```diff
@@ -0,0 +1,71 @@
+# Does not inherit from simplemob
+- type: entity
+  name: Holoparasite
+  id: MobHoloparasite
+  description: A mezmirising whirl of hard-light patterns weaves a marvelous, yet oddly familiar visage. It stands tall, tuning into it's owner's life to sustain itself.
+  components:
+  - type: GhostTakeoverAvailable
+    makeSentient: true
+    name: Holoparasite
+    description: Listen to your owner. Don't tank damage. Punch people hard.
+  - type: Input
+    context: "human"
+  - type: PlayerMobMover
+  - type: PlayerInputMover
+  - type: MovementSpeedModifier
+    baseWalkSpeed : 7
+    baseSprintSpeed : 7
+  - type: DamageOnHighSpeedImpact
+    damage:
+      types:
+        Blunt: 5
+    soundHit:
+      path: /Audio/Effects/hit_kick.ogg
+  - type: Sprite
+    drawdepth: Mobs
+    layers:
+    - map: ["enum.DamageStateVisualLayers.Base"]
+      state: tech_basic
+      sprite: Mobs/Aliens/Guardians/guardians.rsi
+  - type: Clickable
+  - type: InteractionOutline
+  - type: Physics
+    bodyType: KinematicController
+    fixtures:
+    - shape:
+        !type:PhysShapeAabb
+          bounds: "-0.40,-0.30,0.40,0.45"
+      mass: 10
+      mask:
+      - Impassable
+      - VaultImpassable
+      - SmallImpassable
+      layer:
+      - Opaque
+  - type: Damageable
+    damageContainer: Biological
+  - type: MobState
+    thresholds:
+      0: !type:NormalMobState {}
+  - type: HeatResistance
+  - type: CombatMode
+  - type: Internals
+  - type: Examiner
+  - type: Speech
+  - type: Pullable
+  - type: UnarmedCombat
+    range: 2
+    arcwidth: 30
+    arc: fist
+    cooldownTime: 0.2
+    arcCooldownTime: 0.2
+    damage:
+      types:
+        Blunt: 20
```

Comment:
As funny as I think it is
A) Giving them 200 dps
B) Giving them 5 hits per second

seem terrible balance wise and network wise.

Reply:
It IS funny

Reply:
I ungad my maths.

It's still critting players in 1 second with no downside which is stupid.

Reply:
is 75 better?